### DISCUSSION TITLE
Expires header of " Thu, 19 Nov 1981 08:52:00 GMT" sent in Craft 3.4-RC

### DISCUSSION THREAD

**Discussion by FreekVR** [craftcms/cms#8771] [Ideas]
### Description
I've been having some issue with slower page requests in Craft 3.4. It doesn't seem to occur in 3.3 as far as I've been able to debug.

It appears to be caused by session_cache_limiter() being set to a value of 'nocache' as is the default in PHP 7.2's php.ini.

It seems Yii has a workaround for this in its HttpCache class: https://www.yiiframework.com/doc/guide/2.0/en/caching-http#session-cache-limiter

However, I couldn't find any usage of this class inside Craft, so it appears to simply send the "vanilla" php value for session_cache_limiter, which then leads to these cache headers being set:
`Expires: Thu, 19 Nov 1981 08:52:00 GMT Cache-Control: no-store, no-cache, must-revalidate, post-check=0, pre-check=0 Pragma: no-cache`
Ref: https://www.php.net/manual/en/function.session-cache-limiter.php

Suggestion:
- Find out appropriate usage of Yii's HttpCache on Craft requests
OR
- Set an alternative default for session_cache_limiter() eg. allow override from craft config?

### Steps to reproduce
1. Request a page in Craft 3.4-RC3
2. The expires header is set to Thu, 19 Nov 1981 08:52:00 GMT

### Additional info

- Craft version: 3.4-RC3
- PHP version: 7.2
- Database driver & version: n/a
- Plugins & versions: n/a

**Comment by seandelaney** [comment]
I'm having the same issue with one of my Craft projects. Did you ever find an answer and/or resolve the issue?
